backends/arm/passes/arm_pass_utils.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Arm Limited and/or its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+
+import torch
+
+from executorch.exir.dialects._ops import ops as exir_ops
+from torch._ops import OpOverload
+
+
+def create_node(
+    graph: torch.fx.Graph,
+    op_target: OpOverload,
+    args: tuple = (),
+    kwargs: Optional[dict] = None,
+    quantize: bool = False,
+    q_params: tuple = None,
```

Comment:
executorch/backends/arm/passes/arm_pass_utils.py:21:4 Incompatible variable type [9]: Unable to unpack `None`, expected a tuple.

Reply:
Ack